crates/hdk/Cargo.toml

```
@@ -1,10 +1,10 @@
 [package]
 name = "hdk"
-version = "0.0.1"
-description = "third iteration of the holochain hdk"
+version = "0.0.100-alpha.0"
+description = "The Holochain HDK"
 license-file = "LICENSE_CAL-1.0"
-homepage = "https://github.com/holochain/holochain"
-documentation = "https://github.com/holochain/holochain"
+homepage = "https://github.com/holochain/holochain/crates/hdk"
```

Need to specify the branch. Should go with `main` or `develop`? Assuming we'll merge to `main` after release.

```suggestion
homepage = "https://github.com/holochain/holochain/tree/main/crates/hdk"
```

---

oh, maybe we shouldn't specify it at all then. or maybe we should assume the tag?

---

I'm trying to go with the tag. See the _release.toml_ file in the crate that instructs replacement of this pre/post release.